python/tvm/script/__init__.py

```diff
@@ -14,8 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""TVM Script APIs of TVM Python Package, aimed to support TIR"""
-
-from . import tir
-
-from .parser import ir_module, from_source
+"""TVM Script APIs of TVM Python Package"""
+from .parser import ir, ir_module, parse as from_source, tir
```

Comment:
@junrushao it appears this line, combined with the movement of the old parser from `parser/` to `/parser_v1` means that this PR implicitly switches things over to using the new parser. Am I interpreting this correctly, and was that intentional?